packages/studio-web/src/app/microphone.service.ts

```diff
@@ -33,7 +33,11 @@ export class MicrophoneService {
     return new Promise((resolve, reject) => {
       this.recorderEnded.subscribe(
         (blob) => {
-          resolve(blob);
+          if (blob.size < 2000) {
```

Comment:
we could probably make this even smaller - it would be good to have a comment explaining why it works

Reply:
1500? I struggled to get the size less than 3000 (ie by clicking record/stop record as fast as I could) if the microphone was working